components/Navigation.tsx

```diff
@@ -645,21 +645,267 @@ export default function Navigation() {
       </NavigationSection>
 
       <NavigationSection title="Topics">
-        <NavigationLink href="/topics/certified-devices">
+        <NavigationLink
+          href="/topics/certified-devices"
+          subLinks={
+            <Fragment>
+              <NavigationSubLink href="/topics/certified-devices#hows-it-work">
```

Comment:
```suggestion
              <NavigationSubLink href="/topics/certified-devices#how's-it-work?">
```

Reply:
I didn't realize this before but you actually need to include special characters for the url to actually work.